### ISSUE TITLE
Compatibility with windows 11

### ISSUE THREAD

**Issue by CBenghi** [microsoft/Extensible-Storage-Engine#32]
Hello,
I have just noticed that this code is not aligned with the version of esent.dll shipped with windows 11.
At least some files created with the version of esent.dll shipped with Windows 11 cannot be opened with this code.
Is there a plan to align the code here with what ships with Windows?
Thanks,
Claudio

**Comment by JakeSays** [comment]
@CBenghi Here is a good overview of future plans: https://github.com/microsoft/Extensible-Storage-Engine/discussions/21#discussioncomment-5426090

**Comment by CBenghi** [comment]
Thanks for the heads up, @JakeSays.

@MsftBrettShirley,
thanks for the clarification in #21.

**TLDR**: please release, even sporadically, a version of the code compatible with the latest version deployed across all shipped versions of Windows. 

That would allow us to keep waiting for the open version of ese indefinitely, reducing pressure on you, while keeping the project alive.

**Long form.**

I'm the maintainer of a project relying on ManagedEsent for data storage. 

Our users **rarely** have a need to share content files between themselves, so compatibility issues across versions were infrequent in the past. 

Until now, when issues occurred, we could ask our users to update to the latest Windows 10 patch and that would resolve the problem.

Now this cannot be done because Windows 10 and 11 have diverged in the version of esent.dll, and its not possible to ask all users to upgrade windows version.

If you published a version that was capable of reading the latest files, we could deploy its binaries and prevent versioning issues across platforms in the future.

At the moment this is not an option, because if we deploy the current version of the open source dll, we would not be able to read the contents that users on windows 11 have already produced.

Lacking this, the only path forward that I can see is to deploy some conversion service online based on the latest win 11, and move to another persistence format dropping the use of ese as a backing format, but that would be a lot of pain.

Perhaps it was a mistake on our side to chose esent in this scenario, but with your help we can make it work and remain on a technology that has served us well for many years.

Many thanks,
Claudio
